PeerioTechnologies/peerio-desktop
# fix: prevent upload previews from caching

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/8164/desktop-uploading-file-with-same-name-new-content-shows-old-preview
 
#### Testing instructions  

* Open an image file in some image editor.
* In a chat, upload this file.
* Change the image file and save (under the same name at the same path).
* In a chat, upload this file again — see that preview in the upload dialog contains the new image, not the old one.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
